linux/test/ubuntu2204/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Dockerfile for testing the OMERO Linux installation instructions
+# Not intended for production use
+FROM ubuntu:20.04
+MAINTAINER ome-devel@lists.openmicroscopy.org.uk
+
+ARG OMEROVER=latest
+ARG JAVAVER=openjdk11
+ARG ICEVER=ice36
+ARG PGVER=pg12
```

Comment:
pg14 is the default. 
This variable has no impact since there is option to install anything else implemented

Reply:
Yes, in the specific shell script I just install `postgresql` which for 22.04 is postgres 14. But I set PGVER to 14, just to make it clear.